<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: light grey;</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(232, 232, 232);
border:none;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignLeading</set>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>31</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(193, 190, 196);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 201, 175);
color:white;</string>
    </property>
    <property name="text">
     <string>Contact List</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterbutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(232, 232, 232);</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 201, 175);
color:white;</string>
     </property>
     <property name="text">
      <string>Add new profile</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
    </widget>
    <widget class="QPushButton" name="okbutton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="namebox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Name (no space)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="addressbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Adress</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="citybox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Number: 0xxxxxxxxx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QWidget" name="infowindow" native="true">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>281</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 201, 175);
color:white;</string>
      </property>
      <property name="text">
       <string>Profile information</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="margin">
       <number>16</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameinfobox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Name (no space)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addressinfobox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cityinfobox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>City</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numberinfobox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Number: 0xxxxxxxxx</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailinfobox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
border:none;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editnamebutton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="editaddressbutton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editcitybutton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editnumberbutton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editemailbutton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="savebutton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebutton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>430</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
      </property>
      <property name="text">
       <string>Delete this profile</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="backinfo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
      </property>
      <property name="text">
       <string>back</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="callbutton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
      </property>
      <property name="text">
       <string>Call</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="editbutton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>430</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0, 198, 158);
color:white;
border:none;</string>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="backadd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="searchbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search here, name or phone number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="profile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
color:rgb(0, 198, 158);
border:none;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Profiles </string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>profile</zorder>
   <zorder>searchbox</zorder>
   <zorder>listView</zorder>
   <zorder>listView_2</zorder>
   <zorder>label</zorder>
   <zorder>ajouterbutton</zorder>
   <zorder>widget_2</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
